src/main.rs

```diff
@@ -34,9 +34,10 @@ fn main() {
 
     let cfg = match config::Config::new() {
         Ok(c) => c,
+        Err(config::ConfigError::CommandLine(e)) => e.exit(),
         Err(e) => {
-            println!("Failed to read config: {}", e);
-            return;
+            error!("Failed to read config: {}\n", e);
```

Comment:
This will make more sense together with forcing the logger to always print errors. Perhaps, warnings as well.